### ISSUE TITLE
How should DSU get its identity?

### ISSUE THREAD

**Issue by zhtaoxiang** [zhehaowang/ndnfit#2]
According to the design, every DSU should also has its own identity, how to do this? Manually? Or some other choices?

**Comment by zhehaowang** [comment]
For now, let's do this manually. 

In the future, I suppose each DSU would have a similar ID request process, where the DSU's operator would ask his DSU to be authorized by a root.

**Comment by zhtaoxiang** [comment]
Agree, I will keep this issue open. And we should add this process later.
